suite/tests/client-interface/drreg-test.c

```diff
@@ -310,10 +292,7 @@ GLOBAL_LABEL(FUNCNAME:)
         jmp      epilog3
      epilog3:
         add      REG_XSP, FRAME_PADDING /* make a legal SEH64 epilog */
-        pop      REG_XDI
-        pop      REG_XSI
-        pop      REG_XBP
-        pop      REG_XBX
+        POP_CALLEE_SAVED_REGS()
         ret
 #elif defined(ARM)
         b        test5
```

Comment:
What about other code: "git grep 'push callee-saved'" alone shows a number of other instances.  If you do not plan to convert them here, do not mark the issue as Fixed and mention the work still to be done in the commit message.

Reply:
I suggest I fix the occurences for x86 and leave the issue open and will add a comment that ARM/AArch64 is still TBD.

Reply:
They have some pseudo macros already: SAVE_PRESERVED_REGS/RESTORE_PRESERVED_REGS
They don't have the real ABI I see in llvm. I suggest to leave it open.